packages/plugin-ext/src/hosted/browser/hosted-plugin.ts

```diff
@@ -585,7 +585,10 @@ export class HostedPluginSupport {
         return () => {
             performance.mark(endMarker);
             performance.measure(name, startMarker, endMarker);
-            const duration = performance.getEntriesByName(name)[0].duration;
+
+            const entries = performance.getEntriesByName(name);
+            const duration = entries.length > 0 ? entries[0].duration : 0;
```

Comment:
reporting 0 will be bogus

Reply:
When a number is expected reporting `NaN` would have side-effects. Followed the [example Vimeo used](https://bugzilla.mozilla.org/show_bug.cgi?id=1511941#c2) and used a value which is a valid number and also below 100 (the check inside `core` for the `performance` API)…

I'm not sure about the implications of returning `NaN`. From my experience that creates more issues than it solves…

Reply:
Hm interestingly it looks like the number is only used in log entries and for some reason `Number.NaN.toFixed(1)` indeed works and reports `NaN`…

Well. I can change that if you want…

The difference would be:
```
[${this.clientId}] ${prefix} of ${pluginCount} took: NaN ms
vs.
[${this.clientId}] ${prefix} of ${pluginCount} took: 0 ms
```

Reply:
> [${this.clientId}] ${prefix} of ${pluginCount} took: 0 ms

Would be confusing? Maybe we can avoid logging all together if a value is NaN?

Reply:
We certainly can… :slightly_smiling_face: